app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -777,15 +777,14 @@ aside.onebox.xkcd .onebox-body img {
     margin-bottom: 0.2rem !important;
   }
 
-  .d-icon-tag {
-    width: 0.75rem;
-    padding-top: 0.3rem;
-    position: absolute;
-    color: var(--primary-medium);
-  }
-
-  .discourse-tags .discourse-tag:first-of-type {
-    padding-left: 1rem;
+  .discourse-tags {
```

Comment:
Just played a bit with your PR, indeed this appears to be better :slightly_smiling_face: We just need to add a `vertical-align: bottom;` here though otherwise categories and tags aren’t properly aligned.

Reply:
Sure, good catch, added.